Tools/dev-toolbox.1h.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+# <xbar.title>Dev Toolbox</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Ankit Gaurav</xbar.author>
+# <xbar.author.github>theankitgaurav</xbar.author.github>
+# <xbar.desc>Handy menu bar app to serve as swiss-army knife for devs.</xbar.desc>
+#  <xbar.image>https://user-images.githubusercontent.com/9376908/185213187-c6ad3f67-217c-47a4-9283-10d92b1f2aa9.png</xbar.image>
+# <xbar.dependencies>shell,jq,sed</xbar.dependencies>
+# <xbar.abouturl>https://github.com/theankitgaurav/dev-toolbox</xbar.abouturl>
+
+# Variables as preferences of the app:
+#  <xbar.var>select(APP_ICON="⚙️"): App icon to be shown in the menu app. [⚙️,🛠,DevToolBox]</xbar.var>
+
+export PATH="/usr/local/bin:/usr/bin:$PATH"
```

Comment:
Might want to add the new M1 silicon path for Homebrew:

```suggestion
export PATH="/opt/homebrew/bin:/usr/local/bin:/usr/bin:$PATH"
```